wgpu-types/src/lib.rs

```diff
@@ -2850,7 +2862,16 @@ impl TextureFormat {
             ) => Some(Self::Stencil8),
             (Self::Depth24PlusStencil8, TextureAspect::DepthOnly) => Some(Self::Depth24Plus),
             (Self::Depth32FloatStencil8, TextureAspect::DepthOnly) => Some(Self::Depth32Float),
-            (format, TextureAspect::All) => Some(format),
+            (Self::NV12, TextureAspect::Plane0) => Some(Self::R8Unorm),
+            (Self::NV12, TextureAspect::Plane1) => Some(Self::Rg8Unorm),
+            (format, TextureAspect::All) => {
```

Comment:
```suggestion
            (format, TextureAspect::All) if !format.is_multi_planar_format() => {
```

Then you can just fallthrough to the None